library/platform_util.c

```diff
@@ -63,7 +63,7 @@
 /* The number of iterations after which the seed of the non-cryptographic
  * random number generator will be changed. This is used only if the
  * MBEDTLS_ENTROPY_HARDWARE_ALT option is enabled. */
-#define RAND_SEED_LIFE  100
+#define RAND_SEED_LIFE  10000
```

Comment:
Commit 0962e6b7574d57345b409796d90f0c3f725bcc17 message is too long.

Reply:
Actually, I moved this commit to the previous one